src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const testRepeat = [];
+  const value = [];
+  const resalt = {};
+
+  for (const key in robot) {
+    testRepeat.push(robot[key]);
+    value.push(key);
```

Comment:
you can kust check here if your result contains key that you want to add - if yes - it's duplicate, if not - add it, that's can helps avoid another code after for in loop